docs/admin/accessing-the-api.md

```diff
@@ -92,18 +93,14 @@ variety of uses cases:
       on different machines than the apiserver.  Kubelet uses the Secure Port
       to get their pods, to find the services that a pod can see, and to
       write events.  Credentials are distributed to kubelets at cluster
-      setup time. Kubelets use cert-based auth, while kube-proxy uses token-based auth.
+      setup time. Kubelet and kube-proxy can use cert-based authentication or token-based
+      authentication.
 
 ## Expected changes
 
    - Policy will limit the actions kubelets can do via the authed port.
    - Scheduler and Controller-manager will use the Secure Port too.  They
      will then be able to run on different machines than the apiserver.
-   - Clients, like kubectl, will all support token-based auth, and the
```

Comment:
Isn't it already the case that clients support token-based auth? Localhost potentially going away has more to do with the previous point about the scheduler and controller manager needing it.